content/en/docs/reference/using-api/api-concepts.md

```diff
@@ -704,6 +704,8 @@ An encoded Protobuf message with the following IDL:
   }
 ```
 
+The messages described above can also be found at https://github.com/kubernetes/apimachinery/blob/master/pkg/runtime/generated.proto.
```

Comment:
```suggestion
The messages described above can also be found at https://github.com/kubernetes/apimachinery/blob/master/pkg/runtime/generated.proto/.
```

Reply:
Maybe we can add it as a hyperlink and not just paste the link inline.
Something along the lines of :
```
The message above can also be found in kubernetes [sourcecode](https://github.com/kubernetes/apimachinery/blob/master/pkg/runtime/generated.proto/).
```